src/windows_msvc.rs

```diff
@@ -141,24 +141,11 @@ fn include_windows_10_kits(kit_root: &str) {
 }
 
 fn include_windows_10_kits_impl(kit_root: &str) {
-    const VAR_INCLUDE: &str = "INCLUDE";
-
-    if let Ok(include_root) = fs::read_dir(kit_root.to_string() + r"\Include\") {
-        let mut include = env::var(VAR_INCLUDE).unwrap_or_default();
```

Comment:
Before I even start to think about reviewing this mess, is this branch even entered on your system? If so, does the final env var have any elements in it?

Reply:
I mean, I'm using this in a closed source project, and it fixed the underlying issue that winuser.h doesn't get found.

I dumped the include path being set, here it is:

`C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\include;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\ucrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\um;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\cppwinrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\winrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\shared;`